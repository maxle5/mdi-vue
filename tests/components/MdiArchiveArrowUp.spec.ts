
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiArchiveArrowUp from "../../src/components/MdiArchiveArrowUp.vue";

test("MdiArchiveArrowUp snapshot", () => {
  const wrapper = mount(MdiArchiveArrowUp, {});
  expect(wrapper.html()).toMatchSnapshot();
});
