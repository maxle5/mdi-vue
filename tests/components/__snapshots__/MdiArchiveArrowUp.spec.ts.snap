// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiArchiveArrowUp snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M4 21H20V8H4M14 15V18H10V15H7L12 10L17 15M3 3H21V7H3" style="fill: currentColor;"></path>
</svg>"
`;
